packages/app-store/qr_code/api/add.ts

```diff
@@ -4,7 +4,6 @@ import { createDefaultInstallation } from "../../_utils/installation";
 import appConfig from "../config.json";
 
 const handler: AppDeclarativeHandler = {
-  // Instead of passing appType and slug from here, api/integrations/[..args] should be able to derive and pass these directly to createCredential
```

Comment:
Removed this comment as it's not trivial to do it. So, we will keep it as is for now.